spec/models/form_spec.rb

```diff
@@ -247,9 +247,9 @@ def answer_local_authority(lettings_log)
       expect(form.sections[0].class).to eq(Form::Sales::Sections::Setup)
       expect(form.subsections.count).to eq(1)
       expect(form.subsections.first.id).to eq("setup")
-      expect(form.pages.count).to eq(16)
+      expect(form.pages.count).to eq(18)
       expect(form.pages.first.id).to eq("organisation")
-      expect(form.questions.count).to eq(17)
+      expect(form.questions.count).to eq(19)
```

Comment:
I'm scared these tests are going to become like that hated FormHandler test that I eventually killed.

Reply:
Hah, hopefully not, cause it's just the setup section